modules/jena/src/test/scala/ch/epfl/bluebrain/nexus/rdf/jena/Fixture.scala

```diff
@@ -0,0 +1,75 @@
+package ch.epfl.bluebrain.nexus.rdf.jena
+
+import ch.epfl.bluebrain.nexus.rdf.{Graph, Iri}
+import ch.epfl.bluebrain.nexus.rdf.syntax.all._
+import ch.epfl.bluebrain.nexus.rdf.Node.Literal
+import ch.epfl.bluebrain.nexus.rdf.Node.Literal.LanguageTag
+import ch.epfl.bluebrain.nexus.rdf.Vocabulary.{rdf, schema, xml}
+import org.apache.jena.rdf.model.{AnonId, Model, ModelFactory, Resource, ResourceFactory}
+import org.apache.jena.rdf.model.impl.ResourceImpl
+
+object Fixture {
```

Comment:
Are there tests for default Jena mapper values? like `schema:Date`, `xsd:Long`, etc:
```scala
      val jenaLiteral =Literal("1999-04-09T20:00Z", url"http://schema.org/Date".value). asJena
      jenaLiteral.getLexicalForm shouldEqual "1999-04-09T20:00Z"
      jenaLiteral.getDatatypeURI shouldEqual "http://schema.org/Date"
```

Reply:
there are some tests that assert that the lexical form and the datatype are propagated properly (see above comment). The example in your comment tests this exact thing (datatype and lexicalform are propagated correctly).